### PR TITLE
Add option hideCursor

### LINKED ISSUE #2366: hideCursor
Hello, I saw that there is a hideCursor() function but it doesn't work.

Is it possible to have an example to use it?
Thanks in advance.

### PR DESCRIPTION
**Pull request by entonbiba**
Add option `hideCursor`.
- hides the cursor if set to true, by default it will always show.

fixes #2366

### CODE REVIEW

File: src/drawer.js [hunk 1]
```diff
@@ -69,6 +69,7 @@ export default class Drawer extends util.Observer {
         if (this.params.fillParent || this.params.scrollParent) {
             this.style(this.wrapper, {
                 width: '100%',
+                cursor: this.params.hideCursor ? 'hidden' : 'auto',
```
Comment (katspaugh) [inline src/drawer.js:72]: As per [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/cursor), It seems it should be `none` instead of `hidden`?

**Comment by katspaugh** [comment]
Please don't forget to add it [in the docs](https://wavesurfer-js.org/docs/options.html) afterwards.

File: src/wavesurfer.js [hunk 2]
```diff
@@ -270,6 +270,7 @@ export default class WaveSurfer extends util.Observer {
         forceDecode: false,
         height: 128,
         hideScrollbar: false,
+        hideCursor: false,
```
Comment (thijstriemstra) [inline src/wavesurfer.js:273]: also needs to be documented at top of this file.

and changelog entry is missing.

**Review by thijstriemstra — CHANGES_REQUESTED** [review]

File: src/drawer.js (hunk 1 above)
Comment (entonbiba) [reply src/drawer.js:72]: yes

**Comment by entonbiba** [comment]
updated,

**Comment by entonbiba** [comment]
> Please don't forget to add it [in the docs](https://wavesurfer-js.org/docs/options.html) afterwards.

ready once this is merged
#2372

**Review by katspaugh — APPROVED** [review]
Cool, thanks!

**Review by thijstriemstra — APPROVED** [review]

**Pull request merged**